src/domain/rules/entities/SynchronizationRule.ts

```diff
@@ -174,6 +175,10 @@ export class SynchronizationRule {
         return this.syncRule.lastExecuted ? new Date(this.syncRule.lastExecuted) : undefined;
     }
 
+    public get lastExecutedBy(): string | undefined {
+        return this.syncRule.lastExecutedBy?.name || undefined;
```

Comment:
We would normally return directly { id: string; name: string } instead of string